### ISSUE TITLE
cannot find x86_64-w64-mingw32

### ISSUE THREAD

**Issue by FinPl** [JuliaLang/PackageCompiler.jl#403]
Hello, 

I can't  compile the example App using Julia 32 bits version on windows 10. 
Is there a way to add the right compiler to the artifact.toml file?

I spent some time trying to figure out where the problem was. Right now I suppose the get_compiler function should point to the i686 binaries in order for my app to compile.

Any  clue?  Thx!!

**Comment by FinPl** [comment]
Hi again, I finally managed to add the i686 compiler to the artifacts on a local version of PackageCompiler. Everything works as expected afterwards.

**Comment by nielsls** [comment]
Hi - any plans to add support for 32 bit on Windows in the official PackageCompiler?
@FinPl Would you care to just briefly elaborate on what you did to make it work? Thanks!

**Comment by FinPl** [comment]
Hello, the lines 102-104 from PackageCompiler.jl go as follows:

if Sys.iswindows()
        return joinpath(Pkg.Artifacts.artifact"x86_64-w64-mingw32", "mingw64", "bin", "gcc.exe")
    end

This needs to point to the right compiler. I managed to add the references needed in the Artifacts.toml and to get my hands on the i686 version of mingw-w32. I found out they are hosting only the 64bit compiler and I had to download the binary and add the artifact myself. 

Hope I got you on the right direction, best regards.
